docs/api-guide.md

```diff
@@ -216,177 +231,191 @@ Set<String> namespaces = admin.getNamespaceNames();
 You can get the tables of a namespace as follows:
 
 ```java
-// Get the tables of the namespace "ns"
+// Get the tables of the namespace "ns".
 Set<String> tables = admin.getNamespaceTableNames("ns");
 ```
 
-### Get a table metadata
+### Get table metadata
 
-You can get a table metadata as follows:
+You can get table metadata as follows:
 
 ```java
-// Get a table metadata of "ns.tbl"
+// Get the table metadata for "ns.tbl".
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
 
-### Operations for Coordinator tables
+### Specify operations for coordinator tables
 
-Depending on the transaction manager type, you need to create coordinator tables to execute transactions. 
-The following items describe the operations for the coordinator table.
+Depending on the transaction manager type that you are using, you might need to create coordinator tables to execute transactions. In addition to creating tables, you can truncate and drop coordinator tables.
 
-#### Create Coordinator tables
+#### Create coordinator tables
```

Comment:
I guess it should be capitalized because Coordinator is a kind of proper noun in ScalarDB. But, we should discuss it with @brfrn169 and @feeblefakie.

Reply:
Thank you for bringing this up!

After discussing this with @feeblefakie and @brfrn169, we agreed that Coordinator is a component in ScalarDB and should therefore be capitalized.